### PR TITLE
Cleanup/state dropdown

### PR DESCRIPTION
**Pull request by baileydunning**
## Is this a fix or a feature?
Both

## What is the change?
+ Finish unit test for `StateDropdown`
+ Add integration tests in `MainNav`
+ Add styling for `StateDropdown`

## What does it fix?
+ Some styling issues

## Where should the reviewer start?
+ `StateDropdown` - .tsx, .test.js, .scss

## How should this be tested?
+ In browser & `npm test`

### Screenshot
Screen Shot 2021-01-12 at 11 56 18 AM

@saraho1123
@jaypeasee
@baileydunning

### CODE REVIEW

File: src/MainNav/MainNav.test.js [hunk 1]
```diff
@@ -24,17 +29,39 @@ describe('MainNav', () => {
...
+    
+    expect(allButtons.length).toBe(2)
+    expect(screen.getByText('Voyageurs')).toBeInTheDocument()
+  })
+
+  it('should filter park buttons by state and name search', () => {
+    const colorado = screen.getByText('Colorado')
+    
+    userEvent.selectOptions(stateDropdown, [colorado])
+    userEvent.type(searchInput, 'rock')
```
Comment (jaypeasee) [inline src/MainNav/MainNav.test.js:60]: Great integration test here!

File: src/MainNav/MainNav.test.js [hunk 2]
```diff
@@ -24,17 +29,39 @@ describe('MainNav', () => {
...
   it('should be able to filter the list of park buttons based on name match', () => {
-    const searchInput = screen.getByPlaceholderText('search by name')
     userEvent.type(searchInput, 'grand')
     const allButtons = screen.getAllByRole("button")
+    
     expect(allButtons.length).toBe(3)
     expect(screen.getByText('Grand Canyon')).toBeInTheDocument()
   })
+
+  it('should filter park buttons based off state selection', () => {
```
Comment (saraho1123) [inline src/MainNav/MainNav.test.js:46]: Great job problem solving this! I loved it!

File: src/StateDropdown/StateDropdown.scss [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+fieldset {
+  @include flex-column-center;
+  width: 100%;
+  border: none;
+  padding: 0;
+}
+
+select {
+  height: 50px;
+  padding: 0 4%;
```
Comment (jaypeasee) [inline src/StateDropdown/StateDropdown.scss:19]: This input looks great now!

**Review by jaypeasee — APPROVED** [review]
Awesome work here! Let's make the page responsive now.

File: src/StateDropdown/StateDropdown.test.js [hunk 4]
```diff
@@ -14,14 +15,18 @@ describe('StateDropdown', () => {
...
 
   it('should render a dropdown form', () => {
     expect(dropdown).toBeInTheDocument()
   })
 
   it('should select a state with the associated value', () => {
-    userEvent.click(screen.getByText('Minnesota'))
-    // need to figure out how to do the user event for a dropdown form
+    const minnesota = screen.getByText('Minnesota')
+    userEvent.selectOptions(selectForm, [minnesota])
```
Comment (saraho1123) [inline src/StateDropdown/StateDropdown.test.js:27]: It was great to learn about more user events!

File: src/StateDropdown/StateDropdown.tsx [hunk 5]
```diff
@@ -1,23 +1,25 @@
+import './StateDropdown.scss'
+
 type StateDropdownProps = { setStateSelect: (stateSelection: string) => void }
 
 const StateDropdown: React.FC<StateDropdownProps> = props => {
   const { setStateSelect } = props as StateDropdownProps
-
-  const handleSelect = (event: any) => {
+
```
Comment (saraho1123) [inline src/StateDropdown/StateDropdown.tsx:7]: This was so awesome Bailey! Great job getting this to work!

**Review by saraho1123 — APPROVED** [review]
We are so close to being done!

**Pull request merged**